frontend/src/fixtures/commonsFixtures.js

```diff
@@ -23,7 +29,18 @@ const commonsFixtures = {
             "endDate": "6/11/2021",
             "totalPlayers": 50,
             "cowPrice": 15,
+            "milkPrice": 4,
+            "startingBalance": 200,
+            "startingDate": "6/11/2021",
         }
+        //{
+        // "id": 1,
+        // "name": "Anika's Commons",
+        // "startingDate": 5,
+        // "endDate": "6/11/2021",
+        // "cowPrice": 50,
+        // "milkPrice": 15,
+        //}
```

Comment:
Why is this commented out code here?

We generally avoid committing commented out code to the main branch unless there is a compelling reason, and a clear explanation.

Reply:
https://github.com/ucsb-cs156-w22/team04-w22-5pm-HappyCows/blob/fbd2211a027174874520f696edabd474d69e70cc/frontend/src/fixtures/commonsFixtures.js#L25-L43

I commented out the lines and added another fixture with similar values and additional accessors. They were only commented out so that I could revert the change if needed. The current fixtures work, so the commented out lines aren't really needed. 

Would it be best to remove them?

Reply:
I've removed commented code in this file and other files in the branch.